### PR TITLE
Modified the challenge description for better understanding

### LINKED ISSUE #8748: Condense arrays with reduce: Better variable naming
Challenge [Condense arrays with reduce](https://www.freecodecamp.com/challenges/condense-arrays-with-reduce#?solution=%0Avar%20array%20%3D%20%5B4%2C5%2C6%2C7%2C8%5D%3B%0Avar%20singleVal%20%3D%200%3B%0A%0A%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%0AsingleVal%20%3D%20array.reduce(function(total%2C%20currentVal)%7B%0A%20%20return%20total%20%2B%20currentVal%3B%20%2F%2F%20previousVal%20is%20the%20accumulator%0A%7D)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

This is not quite a issue, but i think that if the variable "previousVal" is changed to "total" or "sum" would help others better understand and avoid confusion. As in this case previousVal represents an accumulator.

My code:
```javascript

var array = [4,5,6,7,8];
var singleVal = 0;

// Only change code below this line.

singleVal = array.reduce(function(total, currentVal){
  return total + currentVal; // previousVal is the accumulator
});

```

### PR DESCRIPTION
**Pull request by sahilkhurana19**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #8748 
#### Description

Replaced <code>previousVal</code> with <code>prevTotal</code> as discussed for better understanding of the example code and easier implementation of the function.

### CODE REVIEW

**Comment by atjonathan** [comment]
cc/ @FreeCodeCamp/issue-moderators 
Is this a change we want to make?

**Comment by erictleung** [comment]
@sahilkhurana19 @atjonathan I'm going to comment on the issue #8748 itself.

**Comment by raisedadead** [comment]
Awaiting confirmation on the original issue thread.

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -350,6 +350,7 @@
       "description": [
         "The array method <code>reduce</code> is used to iterate through an array and condense it into one value.",
         "To use <code>reduce</code> you pass in a callback whose arguments are an accumulator (in this case, <code>previousVal</code>) and the current value (<code>currentVal</code>).",
+        "The accumulator is like a total that reduce keeps track of after each operation. The current value is just the next element in the array you're iterating through.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]:
```diff
-reduce
+<code>reduce</code>
```

**Comment by raisedadead** [comment]
@sahilkhurana19 thanks for the changes, just a slight change and we are good to go. Thanks again.

**Pull request merged**